### PR TITLE
Bitwise operator overloading 

### PR DESCRIPTION
**Pull request by Mugdha-Hazra**
Added C++ code for issue #1701 Bitwise Operator Overloading, @tarun26091999 @anubhavitis @veenit1802 @ssaksham @malavikarajeshvikraman @Rohan27s, @rish-singhal sir can you please review my code.

### CODE REVIEW

**Comment by Mugdha-Hazra** [comment]
Hello, @Rohan27s sir this time I had done everything in the right way can you please review it once again and I'm really sorry for the inconvenience in the last pull request.

File: Code/C++/Bitwise_operator_overloading.cpp [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    cout << "NUMBER1 ^ NUMBER2 ";
+    b5.show();
+    cout << "NUMBER1 >> NUMBER2 ";
+    b6.show();
+    return 0;
+}
+
+/*
+      Time Complexity: O(1) 
+
```
Comment (krishna-NIT) [inline Code/C++/Bitwise_operator_overloading.cpp:89]: Mention Space Complexity

**Comment by Mugdha-Hazra** [comment]
@krishna-NIT sir done

File: Code/C++/Bitwise_operator_overloading.cpp (hunk 1 above)
Comment (Mugdha-Hazra) [reply Code/C++/Bitwise_operator_overloading.cpp:90]: @krishna-NIT sir done

**Comment by krishna-NIT** [comment]
resolve the conflict

**Comment by krishna-NIT** [comment]
> resolve the conflict

@Mugdha-Hazra

**Pull request closed without merging**